spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/dataAdapter/GeoPackageAdapter.java

```diff
@@ -93,6 +96,7 @@ public void call(final Subscriber<? super SCDataAdapterStatus> subscriber) {
                  if (!manager.exists(scStoreConfig.getName())) {
                      final URL url = theUrl;
                      Log.d(LOG_TAG, "Attempting to download geopackage from " + url);
+                     store.setStatus(SCDataStoreStatus.SC_DATA_STORE_DOWNLOADING);
```

Comment:
Maybe reconsider the need for changing mutable state of parent class?  Couldn't we just subscribe to `SCStoreStatusEvent`s?